app/src/main/java/org/traccar/client/TrackingController.java

```diff
@@ -100,7 +100,8 @@ public void onPositionUpdate(Position position) {
 
     @Override
     public void onNetworkUpdate(boolean isOnline) {
-        StatusActivity.addMessage(context.getString(R.string.status_connectivity_change));
+        int msgId = isOnline? R.string.status_connectivity_change_online: R.string.status_connectivity_change_offline;
```

Comment:
Didn't notice this before. Please use full words (e.g. `messageId` or just `message`).